big5/operations/default.json

```diff
@@ -636,6 +636,77 @@
         }
       }
     },
+    {
+      "name": "range-agg-1",
+      "operation-type": "search",
+      "index": "{{index_name | default('big5')}}",
+      "body": {
```

Comment:
Is there a reason why we don't include `"request-timeout": 7200,` in `range-agg-1`?

Reply:
Good catch. This 2 range aggregation operations are actually copied from existing `range-auto-date-histo` and `range-auto-date-histo-with-metrics`. I think `"request-timeout": 7200` is just needed when the request is by default taking long time. So I will remove this from range-agg-2